recipes/hdbscan/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.8" %}
+
+package:
+  name: hdbscan
+  version: {{ version }}
+
+source:
+  fn: hdbscan-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/h/hdbscan/hdbscan-{{ version }}.tar.gz
+  sha256: ea09e8ad60a1734e46000e71cc4b4ca274629e5ed9c54015cf4d4150c1238a17
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scikit-learn >=0.16
+    - cython >=0.17
+
+  run:
+    - python
+    - scikit-learn >=0.16
+
+test:
+  imports:
+    - hdbscan
+
+  requires:
+    - nose
```

Comment:
Ugh. Yes, sorry. I'm still learning how to make conda recipes.

  python setup.py test

would be the tests I would like to run -- how exactly do I add that to the
tests as well as the imports? Sorry for my lack of ability with conda.

On Fri, Jul 15, 2016 at 5:02 AM, Patrick Snape <notifications@github.com>
wrote:

> In recipes/hdbscan/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/1032#discussion_r70942072>
> :
>
> > +  build:
> > +    - python
> > +    - setuptools
> > +    - scikit-learn >=0.16
> > +    - cython >=0.17
> > +
> > +  run:
> > +    - python
> > +    - scikit-learn >=0.16
> > +
> > +test:
> > +  imports:
> > +    - hdbscan
> > +
> > +  requires:
> > +    - nose
>
> Test requires nose? But we don't seem to be running any tests?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/1032/files/05412c58799d9b3968830a8f1f3e83d7f9341117#r70942072>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALaKBdA5vzXSYp9p3SZKd9mrh1UnXXGVks5qV0yegaJpZM4JLwcq>
> .
>